addons/account/models/account_move.py

```diff
@@ -137,7 +136,10 @@ def post(self):
                         # If invoice is actually refund and journal has a refund_sequence then use that one or use the regular one
                         sequence = journal.sequence_id
                         if invoice and invoice.type in ['out_refund', 'in_refund'] and journal.refund_sequence:
+                            if not journal.refund_sequence_id:
+                                raise UserError(_('Please define a sequence for the refunds'))
```

Comment:
in stable versions, if you add new translated terms you need to modify the .pot file also or they never will appear in transifex.